.gitignore

```diff
@@ -8,35 +8,3 @@ node_modules/
 npm-debug.log
 yarn-error.log
 
-# Xcode
-#
-build/
-*.pbxuser
-!default.pbxuser
-*.mode1v3
-!default.mode1v3
-*.mode2v3
-!default.mode2v3
-*.perspectivev3
-!default.perspectivev3
-xcuserdata
-*.xccheckout
-*.moved-aside
-DerivedData
-*.hmap
-*.ipa
-*.xcuserstate
-project.xcworkspace
-
-# Android/IntelliJ
-#
-build/
-.idea
-.gradle
-local.properties
-*.iml
-
-# BUCK
-buck-out/
-\.buckd/
-*.keystore
```

Comment:
What's the intention behind this removal?

Reply:
Not sure, one of our devs may have removed it @amogh-jrules @chestersim